plivo/rest/client.py

```diff
@@ -100,6 +104,11 @@ def __init__(self, auth_id=None, auth_token=None, proxies=None, timeout=5):
         self.addresses = Addresses(self)
         self.identities = Identities(self)
         self.call_feedback = CallFeedback(self)
+        self.end_users = EndUsers(self)
```

Comment:
This looks very clean and good. Since you added many new resources. Add the UTC also.  This is there in SDK deployment documentation
https://docs.google.com/document/d/1fs1kGEB2XxGUJcSGXDnlmLZjVOgf4T5xC5aLBbxy7LQ/edit

Reply:
Sure will add that @Ajay-KG